test/specs/happy.js

```diff
@@ -177,6 +176,24 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
         verificationComplete.checkBackArrowIsNotDisplayed()
       })
 
+      it('should take one selfie using the camera stream', async () => {
+        goToPassportUploadScreen(`?language=${lang}&async=false`)
+        uploadFileAndClickConfirmButton('passport.jpg')
+        documentUploadConfirmation.takeSelfie()
+        documentUploadConfirmation.confirmBtn.click()
+        verificationComplete.verifyUIElements(verificationCompleteCopy)
+        verificationComplete.checkBackArrowIsNotDisplayed()
+      })
+
+      it('should take multiple selfies using the camera stream', async () => {
+        goToPassportUploadScreen(`?language=${lang}&async=false&useMultipleSelfieCapture=true`)
+        uploadFileAndClickConfirmButton('passport.jpg')
+        documentUploadConfirmation.takeSelfie()
+        documentUploadConfirmation.confirmBtn.click()
+        verificationComplete.verifyUIElements(verificationCompleteCopy)
+        verificationComplete.checkBackArrowIsNotDisplayed()
+      })
+
```

Comment:
How are we confirming if multiple selfies were taken for this test?
Code wise looks pretty similar.

Reply:
The second test uses the `?useMultipleSelfieCapture=true` query string which uses the option to send 2 requests for selfie. The fact that the user can get to the end of the flow means that both requests succeed. If one of the two requests fails the user will see an error. This tests the happy path. In terms of UX/UI nothing changes between standard capture and multiple frame capture.